parse/PythonParser.cpp

```diff
@@ -170,6 +170,7 @@ PythonParser::PythonParser(PythonCommon& _python, const boost::filesystem::path&
         auto py_variable_wrapper = py::class_<variable_wrapper>("__Variable", py::no_init)
             .def(py::self_ns::self & py::other<value_ref_wrapper<double>>())
             .def(py::self_ns::self & py::other<condition_wrapper>())
+            .def(py::self_ns::self | py::other<condition_wrapper>())
```

Comment:
FYI you could use the python data model and define these operators via functions. 

```python
class Or:
    def __or__(self, other):
        print("Or")
        return other


assert 1 == Or() | 1
# >>> Or


# If class does not support operation, the return method of other is called.
class Nor:
    def __or__(self, other):
        print("NOR")
        return NotImplemented


class Ror:
    def __ror__(self, other):
        print("ROR")
        return other

assert Nor() | Ror()
# >>> NOR
# >>> ROR
```

Reply:
Not sure if I need to do this for C++ structures